src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidator.java

```diff
@@ -226,4 +229,35 @@ private Errors validateOnRegisteredInCountryFormedInNotSupplied(TrustCorporateDt
 
         return errors;
     }
+
+    private Errors validateCeasedDate(TrustCorporateDto trustCorporateDto, LocalDate trustCreationDate, Errors errors, String loggingContext) {
+
+       Boolean isStillInvolved = trustCorporateDto.isCorporateBodyStillInvolvedInOverseasEntity();
+       final String qualifiedFieldNameInvolved = getQualifiedFieldName(PARENT_FIELD,
+               TrustCorporateDto.CORPORATE_BODY_STILL_INVOLVED_IN_OVERSEAS_ENTITY);
+
+       if (UtilsValidators.isNotNull(isStillInvolved, qualifiedFieldNameInvolved, errors, loggingContext)) {
```

Comment:
See [this](https://github.com/companieshouse/overseas-entities-api/pull/395/files#r1625931021) comment on another PR. This might also apply here - if we're on the registration journey we don't want to do a null check on the 'still involved' field as I'm guessing it won't be there.

Reply:
https://companieshouse.atlassian.net/browse/UAR-1452 - from ac 5 presumably it will be, the whole ticket would have no purpose.

Reply:
Yep, but as discussed, first line of UAR-1452 says "_**As a** ROE Update/Remove user_" so I don't think this applies to registration. We need to clarify that with Vern but then also look at the existing API validation implementation, as it looks like all the new validation code you've written will run for a registration but not an Update/Remove. The opposite of what's required...

Reply:
Removed the check and updated the unit tests - nothing broke in the dev testing. 

There were three OverseasEntitySubmissionDtoValidatorTest verifications that broke three trust unit tests because the verify tests were expecting the validations not to be called, which they are now being, given that update and remove are no longer excluded. This has now been changed.

Reply:
Also exclude registration journey from verifying the is still involved field.

Reply:
👍 (I've added a new comment related to the testing of this, so will now resolve this one).